contrib/startup_regtest.sh

```diff
@@ -169,7 +169,7 @@ fund_ln() {
 
 	echo "minning into address " $ADDRESS
 
-	bitcoin-cli -regtest generatetoaddress 50 $ADDRESS
+	bitcoin-cli -regtest generatetoaddress 125 $ADDRESS
```

Comment:
I think they always did?  100 is enough though.

Reply:
100 it is!